src/pybind/mgr/dashboard/frontend/src/app/shared/services/summary.service.spec.ts

```diff
@@ -58,7 +60,31 @@ describe('SummaryService', () => {
     })
   );
 
-  it('should get summary', () => {
-    expect(summaryService.get()).toEqual(jasmine.any(Object));
+  it('should call get', () => {
```

Comment:
All this tests can't be run independently. They rely on the tests before this test.
They all depend on `authStorageService.set('foobar');` and `summaryService.refresh();` to be called.
I would suggest putting those 3 tests into their own describe and use a beforeEach to initialize what they need.

Reply:
Done